torchvision/ops/boxes.py

```diff
@@ -16,7 +16,7 @@ def nms(boxes: Tensor, scores: Tensor, iou_threshold: float) -> Tensor:
     to their intersection-over-union (IoU).
 
     NMS iteratively removes lower scoring boxes which have an
-    IoU greater than iou_threshold with another (higher scoring)
+    IoU greater than :attr:`iou_threshold` with another (higher scoring)
```

Comment:
It's not really an attribute, just a parameter. We typically just use double backticks for those, the renderring will be the same:

```suggestion
    IoU greater than ``iou_threshold`` with another (higher scoring)
```

Reply:
I will change that.

Is there no sphinx way to reference a parameter? In my head just writing it there seems error prone if a parameter gets renamed.